src/test/java/com/hyperwallet/clientsdk/util/HyperwalletApiClientTest.java

```diff
@@ -2199,6 +2205,67 @@ public void testPostProxy_withIncorrectURL() {
         }
     }
 
+    @Test
+    public void testGet_exceedReadTimeout() throws InterruptedException {
```

Comment:
is there a test for connection timeout, simulation?

Reply:
Good call. I'm still investigate how to test Socket timeout 
Do you know any approach ?

Reply:
you can target this service https://httpstat.us/